sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/RegisterAppInterfaceResponse.java

```diff
@@ -376,7 +376,11 @@ public void setSystemSoftwareVersion(String systemSoftwareVersion) {
 		setParameters(KEY_SYSTEM_SOFTWARE_VERSION, systemSoftwareVersion);
     }
 
-    public String getSystemSoftwareVersion() {    
-    	 return getString(KEY_SYSTEM_SOFTWARE_VERSION);
+    public String getSystemSoftwareVersion() {
+        return getString(KEY_SYSTEM_SOFTWARE_VERSION);
+    }
+
+    public Boolean getIconResumed() {

```

Comment:
Please add a corresponding `setIconResumed` method to follow the rest of the parameters.